test/helpers/contracts.js

```diff
@@ -73,6 +86,67 @@ const deployUpgradeableContract = async (
   ];
 };
 
+const getLatestVersionFromFs = async contractName => {
+  const dir = path.join(__dirname, '../../contracts/');
+  let latestVersion = '0_0_0';
+  return new Promise(res => {
+    glob(`${dir}/**/?(${contractName}?(V[0-9]*)).sol`, {}, (er, files) => {
+      files.forEach(file => {
+        const [latestMajor, latestMinor, latestPatch] = latestVersion.split(
+          '_'
+        );
+        let [, versionToCompare] = parseContractName(file);
+        versionToCompare = versionToCompare.split('.')[0];
+        const [major, minor, patch] = versionToCompare.split('_');
+        if (parseInt(major, 10) > parseInt(latestMajor, 10)) {
+          latestVersion = versionToCompare;
+        } else if (
+          parseInt(major, 10) === parseInt(latestMajor, 10) &&
```

Comment:
this could just be `major === latestMajor`. Unless you wanted `V1_0_0` to match `V01_0_0`.